fedimint-server/src/config.rs

```diff
@@ -48,6 +48,8 @@ pub struct ServerConfig {
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct ServerConfigPrivate {
+    /// The secret key guardian should use to sign remote requests
+    pub admin_key: SerdeSecret<hbbft::crypto::SecretKey>,
```

Comment:
Why is the secret version of the key in the server config if the server only needs to verify? Also, any particular reason for choosing `hbbft::crypto` over `secp256k1`?

Reply:
In 
[Use seed phrase instead of a single key for admin authentication](https://github.com/fedimint/fedimint/pull/1171/commits/bd8f595a8e788fc0449b05a0984a6d4668806120) I change it to derivable secret.